src/hardware/targets.json

```diff
@@ -1211,9 +1211,45 @@
                 "platform": "esp32",
                 "firmware": "Unified_ESP32_2400_TX",
                 "prior_target_name": "RadioMaster_Zorro_2400_TX"
+            },
+            "ranger": {
+                "product_name": "RadioMaster Ranger 2.4GHz TX",
+                "lua_name": "RadioMstr Ranger",
+                "layout_file": "Radiomaster Ranger.json",
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
+            },
+            "ranger-micro": {
+                "product_name": "RadioMaster Ranger Micro 2.4GHz TX",
+                "lua_name": "RM Ranger Micro",
+                "layout_file": "Ranger Micro.json",
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
+            },
+            "ranger-nano": {
+                "product_name": "RadioMaster Ranger Nano 2.4GHz TX",
+                "lua_name": "RM Ranger Nano",
+                "layout_file": "Ranger Micro.json",
+                "overlay": {
+                    "power_values": [-18,-15,-12,-8,-5,0]
+                },
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
             }
         },
         "rx_2400": {
+            "er5": {
```

Comment:
No platformio target?

Reply:
No we don't need one. Configurator can use the alias, and I'm just adding it to targets for completeness for when configurator can use it 😄

Reply:
Ok. Perfect!